contrib/completions/bash/kubectl

```diff
@@ -431,9 +432,11 @@ _kubectl_get()
     must_have_one_noun+=("secret")
     must_have_one_noun+=("service")
     must_have_one_noun+=("serviceaccount")
+    must_have_one_noun+=("subreplicaset")
```

Comment:
Assuming you are going to remove the subReplicaSet parts, this commit generally looks good. I would like @nikhiljindal to make a final pass for client side changes once this PR is in its final shape (not WIP).